peps/pep-0705.rst

```diff
@@ -399,6 +426,15 @@ Calling the type ``Readonly``
 
 ``Read-only`` is generally hyphenated, and it appears to be common convention to put initial caps onto words separated by a dash when converting to CamelCase. This appears consistent with the definition of CamelCase on Wikipedia: CamelCase uppercases the first letter of each word. That said, Python examples or counter-examples, ideally from the core Python libraries, or better explicit guidance on the convention, would be greatly appreciated.
 
+Reusing the ``Final`` annotation
+--------------------------------
+
+The :class:`~typing.Final` annotation prevents an attribute from being modified, like the proposed ``ReadOnly`` qualifier does for TypedDict items. However, it is also documented as preventing redefinition in subclasses too; from :pep:`591`:
+
+    The ``typing.Final`` type qualifier is used to indicate that a variable or attribute should not be reassigned, redefined, or overridden.
+
+This does not fit with the intended use of ReadOnly. Rather than introduce confusion by having ``Final`` behave differently in different contexts, we chose to introduce a new qualifier.
```

Comment:
```suggestion
This does not fit with the intended use of ``ReadOnly``. Rather than introduce confusion by having ``Final`` behave differently in different contexts, we chose to introduce a new qualifier.
```